corehq/motech/repeaters/views/repeaters.py

```diff
@@ -63,8 +63,11 @@ def page_context(self):
         return {
             'repeaters': self.repeaters,
             'pending_record_count': RepeatRecord.count(self.domain),
-            'gefingerpoken': self.request.couch_user.is_superuser or
-                             toggles.IS_CONTRACTOR.enabled(self.request.couch_user.username)
+            'gefingerpoken': (
```

Comment:
The template shows the Data Forwarder's "Configure" button if "[gefingerpoken][1]" is set.

  [1]: https://en.wikipedia.org/wiki/Blinkenlights

Reply:
haha, that's cute.  An in-line comment would help make this readable if you're committed to the reference.